examples/gno.land/r/gov/dao/types.gno

```diff
@@ -0,0 +1,232 @@
+package dao
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+)
+
+type ProposalID int64
+
+func (pid ProposalID) String() string {
+	return ufmt.Sprintf("%v", int64(pid))
+}
+
+// VoteOption is the limited voting option for a DAO proposal
+// New govDAOs can create their own VoteOptions if needed in the
+// future.
```

Comment:
Are you expecting the upcoming govdaos to be r/gov/dao/v2? Do you anticipate them to "import r/gov/dao(/v1)" (the current package)?

I think it's good to use suffixes for things that will receive upgrades, such as /v1 instead of relying on an implicit v1.

Reply:
r/gov/dao is the proxy realm, the global, immutable entry point. Implementations live in their version folder (v3 being the first one, and then followed by v4,v5, and so on).